### ISSUE TITLE
Capítulo 1 terminado

### ISSUE THREAD

**Issue by fbitti** [btcbrdev/dominandobitcoin-site#6]
Acabo de concluir a tradução do capítulo 1 e está à vossa disposição para revisão, principalmente se o texto é claro e soa brasileiro.

Alguns pontos:
- baixei o Multibit e posso colocar as capturas de tela em português. Prefiro deixar isso para outro momento quando estivermos organizando o conteúdo graficamente.
- Nomes de pessoas: acho que Alice deveria continuar Alice, não Alícia, pois ambos existem em português e deveríamos evitar modificar sem necessidade.
- temos decidir como colocar os exchanges brasileiros na lista, e quais. Quero dizer, colocamos no corpo do texto e informamos que não estava no original, ou criamos um apêndice à tradução?
- Qual capítulo posso continuar? Continuo com o 2 se ninguém se opuser.

É isso, colegas!
Vamos que vamos

Fernando

**Comment by Anderson-Juhasc** [comment]
Perfeito @fbitti !
- Sobre as imagens eu posso cuidar delas durante as revisões.
- O nome Alice eu acho válido.
- Podemos entrar em contato com todas as exchanges brasileiras e pedir uma doação/investimento para ser divido entres os colaboradores do livro e podemos sim fazer a apêndice da tradução para falar sobre as mudanças existentes.
- Pode pegar o capítulo 2 sim.

Obrigado pelo grande ajuda!

**Comment by fbitti** [comment]
Vcs sabem como o nome de todos que estamos contribuindo aparece, tal como no github original?
https://github.com/aantonop/bitcoinbook/graphs/contributors

**Comment by paladini** [comment]
Perfeito @fbitti , concordo em todos os pontos com o @Anderson-Juhasc . Eu vou revisar o capítulo 1 em breve, assim como revisar o prefácio feito pelo @Anderson-Juhasc . Boa sorte no começo do capítulo 2.

**Comment by paladini** [comment]
Quem mais pode contribuir com a revisão do capítulo 1?

**Comment by Anderson-Juhasc** [comment]
@fbitti eles estão contribuindo via GIT direto no repositório e assim mostra os gráfico de contribuições, talvez poderemos criar um repositório separado para a wiki do livro, acho que mostrará os gráficos das contribuições.

Se você for em revisões no cabeçalho vc pode ver as alterações e contribuições:
https://github.com/btcbrdev/dominandobitcoin/wiki/_history

**Comment by fbitti** [comment]
@Anderson-Juhasc confesso que eu não me importo em receber $$$ pelo trabalho de tradução, mas gostaria muito de que as contribuições aparecessem como ali. É meio vaidade mas é um  reconhecimento muuuito bacana. Por que razões decidimos não contribuir direto no repositório?

Outra coisa, se eu revisar o que o @rhcastilhos fez no capítulo 10 e mudar uma frase e ele não estiver de acordo, ele pode reverter? Me dá a sensação de que os revisores cada um edita direto sobre a wiki e não ficam mapeados onde foi nem o que tinha antes!

Obrigado a todos! Estou aprendendo muito com esse time de feras!

**Comment by Anderson-Juhasc** [comment]
@fbitti certo, podemos escolher algumas exchanges para colocar e deixamos o endereço de doação no README do repositório para quem se interessar em doar algo.

No repositório pensei em colocar o site, não tinha imaginado que a wiki não mostrava esses dados, se achar melhor podemos fazer só o markdown nesse repositório e fazer um repositório a parte para o site do livro.

Outro motivo de usamos a wiki foi praticidade para contribuir sem conhecimento de git, se bem que da para usar a interface gráfica também.

O que acham melhor?

**Comment by Anderson-Juhasc** [comment]
Copiei o repositório da wiki e botei nesse novo repositório: https://github.com/btcbrdev/dominandobitcoin-livro agora da para ver as mudanças nos commits.

Mudamos o livro para esse novo repositório e usamos o atual para o site?

**Comment by Anderson-Juhasc** [comment]
Outra coisa, sobre ser pago por esse trabalho, acho que pagando um café ou um almoço para os envolvidos pode gerar um estimulo a mais dos mesmos.

**Comment by fbitti** [comment]
Oi @Anderson-Jusach, não sei o q é melhor, já Q vc comenta que no wiki se evita ter de aprender Git, como dizem, cuidado com o que se pede? Rsrs 

Sobre ser pago, concordo contigo.  Só dizia que no meu caso não é meu motivador, mas de nenhuma forma falo por todos.  Mas claro Q aceito. ;)

**Comment by Anderson-Juhasc** [comment]
@fbitti o @rhcastilhos já continuou dentro do novo repositório, creio que não seria um problema continuar o livro nele e deixar esse para o website. Eu não sei criar novos arquivos pela interface do website, mas se for um problema é só pedir que criamos mais um, e para editar acredito que seja parecido como estava fazendo na wiki só que você coloca um título e um comentário para explicar sua alteração.

Acho que isso pode ajudar: http://rogerdudler.github.io/git-guide/index.pt_BR.html

**Comment by paladini** [comment]
Galera, postei essa mesma issue lá no projeto de tradução no GitHub! :) 

https://github.com/btcbrdev/dominandobitcoin-livro/issues/2
